packages/fiori-freestyle-writer/src/packageConfig.ts

```diff
@@ -28,21 +28,27 @@ export function getPackageJsonTasks({
     startFile?: string;
     localStartFile?: string;
 }): { start: string; 'start-local': string; 'start-noflp': string; 'start-mock'?: string } {
-    const sapClientParam = sapClient ? `?sap-client=${sapClient}` : '';
-    let urlParam = [sapClientParam, 'sap-ui-xx-viewCache=false'].filter((param) => !!param).join('&');
-    urlParam = urlParam ? `?${urlParam}` : '';
-    const params = `${urlParam}${flpAppId ? `#${flpAppId}` : ''}`;
+
+    const addParams = [['sap-ui-xx-viewCache', 'false']];
```

Comment:
Does `addParams` stand for additional parameters?

Reply:
Sure...or params to be added :)